src/scribe_data/wikidata/query_data.py

```diff
@@ -234,11 +234,10 @@ def query_data(
 
                 results_final.append(r_dict)
 
+            path = os.path.dirname(q)
```

Comment:
I'm going to revert these changes for now, @SethiShreya, but will reference them in the future! Ideally we would have formatting files for all of the languages anyway, and this is coming once I'm done reworking the formatting process :)

Thanks for thinking of this, and again I'll reference this later 😊

Reply:
Checking the formatting file was just a small part, the main task these PR was doing is to run those bifurcated queries like Gurmukhi and Shahmukhi in Punjabi, did you reviewed it, and if it is not important right now then we will surely work on that later, that's not an issue :)